doc: Update migration docs post TTN V2 shutdown (#676)
TheThingsIndustries/lorawan-stack-docs: doc/content/getting-started/migrating/faq.md

### Why should I migrate my devices and gateways to {{% tts %}}? 

> {{% tts %}} is more scalable, more secure, and supports more of the LoRaWAN specification than {{% ttnv2 %}}. The Things Network {{% ttnv2 %}} machines have been completely shut down on **December 8, 2021**. Only private The Things Industries {{% ttnv2 %}} deployments might still be running, but the associated software will no longer be developed or maintained by The Things Industries.

Reviewer: ```suggestion
{{% tts %}} is more scalable, more secure, and supports more of the LoRaWAN specification than {{% ttnv2 %}}. The Things Network {{% ttnv2 %}} machines have been completely shut down on **December 8, 2021**. Some private The Things Industries {{% ttnv2 %}} infrastructure is still be running, but the associated software will no longer be developed or maintained by The Things Industries.
```

### When should I migrate my devices and gateways to {{% tts %}}? 

> There is no migration deadline for private {{% ttnv2 %}} deployments to migrate, but since there is no official support anymore, the sooner the better.

Reviewer 2: I think that the `Start migrating` message should stay. We want to relay the importance of this action.
